### PR TITLE
config-file: bail if it doesn't exist and correctly mount if it does

### PR DESCRIPTION
**Pull request by nwmcsween**
untested

### CODE REVIEW

**Comment by wyrie** [comment]
Thanks for the github action! I'd like to use it but I am getting the `unknown flag: --ci-config` error which I believe is fixed in this PR.

Can you merge and bump please?

**Review by aairey — APPROVED** [review]
The flag `--ci-config` should come behind the image name in the docker run command.
This PR fixes that.

File: src/main.ts [hunk 1]
```diff
@@ -61,16 +63,17 @@ async function run(): Promise<void> {
...
-        '--mount',
-        `type=bind,source=${configFile},target=/.dive-ci`,
-        '--ci-config',
-        '/.dive-ci'
-      )
+    const cmdOptions = []
+    
+    if (configFile) {
+      runOptions.push('-v', `${configFile}:/.dive-ci`)
+      cmdOptions.push('--config-file', '/.dive-ci')
```
Comment (leemeador) [inline src/main.ts:70]: Should this not use the yaml file supplied in the parameters?

Comment (leemeador) [reply src/main.ts:70]: Nevermind. I see that the config file is mapped to this name inside the container..

**Comment by leemeador** [comment]
Your fork does not have the /dist/index.js file rebuilt from the your /src/index.ts so when I use your fork, it fails still

**Comment by MaxymVlasov** [comment]
@yuichielectric  could you please merge it? v0.0.4 is broken

**Comment by MaxymVlasov** [comment]
Just in case, #373 works

**Pull request closed without merging**